templates/partials/receipts-table.html

```diff
@@ -1,11 +1,11 @@
 <table id="results" class="data-table" data-type="donation">
   <thead>
     <tr>
-      <th scope="col">Contributor Name</th>
+      <th scope="col">Contributor name</th>
       <th scope="col">State</th>
       <th scope="col">Employer</th>
       <th scope="col">Amount</th>
-      <th scope="col">Date</th>
+      <th scope="col">Receipt date</th>
```

Comment:
So, I checked and that column is correct, we are sorting on the day the receipt was made- "Receipt date" 

Like I said, the column names are confusing and I am up to tweaking them. Because it is confusing, I think it is important to be explicit about which date we are showing, rather than just saying date